_locales/zh-CN/messages.json

```diff
@@ -0,0 +1,157 @@
+{
+	"name":
+	{
+		"message": "搁置的标签页",
+		"description": "Extension name. Displayed in the manifest and pane header"
+	},
+	"description":
+	{
+		"message": "为Chromium浏览器提供旧版 Microsoft Edge 中\"搁置标签页\" 功能",
+		"description": "Extension description"
+	},
+	"author":
+	{
+		"message": "Michael \"XFox\" Gordeev",
+		"description": "Author name"
+	},
+	"options":
+	{
+		"message": "设置",
+		"description": "Alternative text for options button in the pane"
+	},
+	"loadOnRestore":
+	{
+		"message": "在重新打开时加载页面",
+		"description": "Label for option"
+	},
+	"swapIconAction":
+	{
+		"message": "点击拓展图标来搁置所有标签页 (按Alt+P或右键来打开侧栏)",
+		"description": "Label for option"
+	},
+	"github":
+	{
+		"message": "查看GitHub页面",
+		"description": "Link title"
+	},
+	"contributors":
+	{
+		"message": "项目贡献者",
+		"description": "Link title"
+	},
+	"feedback":
+	{
+		"message": "给我们反馈",
+		"description": "Link title"
+	},
+	"buyMeACoffee":
+	{
+		"message": "给我买杯冰红茶!",
+		"description": "Link title"
+	},
+	"credits":
+	{
+		"message": "由Michael 'XFox' Gordeev开发",
+		"description": "Options menu credits"
+	},
+	"setAside":
+	{
+		"message": "搁置当前的所有标签页",
+		"description": "Save collection action name. Used in the pane, extension context menu and manifest shortcuts"
+	},
+	"nothingSaved":
+	{
+		"message": "你目前没有搁置的标签页",
+		"description": "Placeholder for empty pane"
+	},
+	"removeTab":
+	{
+		"message": "从标签集中移除标签页",
+		"description": "Button hint on a tab card"
+	},
+	"restoreTabs":
+	{
+		"message": "恢复标签页",
+		"description": "Collection restore action link name"
+	},
+	"more":
+	{
+		"message": "更多...",
+		"description": "Collections' more button title"
+	},
+	"restoreNoRemove":
+	{
+		"message": "恢复但不移除标签页",
+		"description": "Context action item name"
+	},
+	"removeCollection":
+	{
+		"message": "移除标签集",
+		"description": "Collection remove action name"
+	},
+	"removeCollectionConfirm":
+	{
+		"message": "你确定要移除这个标签集吗?",
```

Comment:
Translation from Chinese: Are you sure you want to remove this tag set?
Correct translation: Are you sure you want to remove this ta**b** set?